### PR TITLE
At homepage

### PR DESCRIPTION
**Pull request by mwakiogeofrey**
The project is about writing Hello microverse word and style it
The project should not have any linters problem

### CODE REVIEW

**Review by mwakiogeofrey — COMMENTED** [review]
Adding modified html and css files

**Comment by mwakiogeofrey** [comment]
How to apply linters

**Comment by Gopxfs** [comment]
Hello @mwakiogeofrey, please keep in mind that each feature should have only one Pull Request. This means that, after you send your code for review and receive a list of requested changes, you should commit your updated code to the same branch. These new changes are going to be displayed here (on the same PR) right under your old commits/code reviewer comments. By doing this, you will ensure that the requested changes were updated, allowing an easier track of your progress. Also, it makes reviews a lot easier 😉.  

> After the PR is approved, you will merge it and create a new one for your next feature.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @mwakiogeofrey 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Implemented some of the changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
Hello @mwakiogeofrey, I've checked that you added the [Microverse README template](https://github.com/microverseinc/readme-template), but looks like something went wrong with the format. To copy it properly, please access [this link](https://github.com/microverseinc/readme-template), click on the README file:
image

Click on the `Raw` button:
image

Copy the whole page:
image

Open your README file, remove everything and paste the template that you copied. Now, check for the comments that Microverse wrote inside the template and follow these instructions. Then, remove the comments and your template is ready 😉.

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @mwakiogeofrey , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.



 ## Highlights
- Linters passing ✔️ 
- Good commit messages ✔️ 
- Correct GitHub flow ✔️ 

 ## Required Changes ♻️
- [x] Good job on using the README template provided by Microverse 👍🏽 but one thing to fix is that your README file is currently on the master branch and it hasn't been edited to match the project so in order to get the approval you should consider these steps:
  - Create a `README.md` file in the branch [at-homepage](https://github.com/mwakiogeofrey/LINTERS-PROJECT/tree/at-homepage) ✍🏽 
  - Copy the content from the README file in master and paste it in your newly created README 📄 
  - Edit the README to add your info and info about your project 📎 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -17,10 +18,38 @@ project by saying hello to microverse and styling the text using css.
...
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
```
Comment (osamaashraf6) [inline README.md:42]: - [ ] As stated by the previous reviewer, Kindly use the Microverse template and update it with your info and consider including all main sections in your readme. You can copy the readme from [here](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) with markdown format and update info.

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi  @mwakiogeofrey ☺️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

you-are-near

## Highlights ✔️
- [x] Good commit messages.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -96,18 +176,27 @@ Feel free to check the [issues page](../../issues/).
...
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p
```
Comment (iambenkis) [inline README.md:202]: - [ ] [**OPTIONAL**] This note is only instructions for you to set up the license. I think you should remove it to keep your README looking professional.

**Review by iambenkis — APPROVED** [review]
Hi @mwakiogeofrey , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**